src/pages/RunEncounter/QuickAddModal.tsx

```diff
@@ -100,6 +107,27 @@ const QuickAddModal: React.FunctionComponent = () => {
                 ></Form.Control>
               </Form.Group>
             </Col>
+            <Col>
+              <Form.Group className="mv-3" controlId="exampleForm.ControlTextArea1">
+                <Form.Label>Player Status</Form.Label>
```

Comment:
I would argue for making this "Creature Type" and then  renaming the modal and button to "Quick Add Creature"